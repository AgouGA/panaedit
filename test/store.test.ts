import { describe, it, expect } from 'vitest'
import createStore from '../src/store/store'

it('should be able to create a store', async ({ expect }) => {
	expect(createStore).toBeDefined()
	expect(createStore).toMatchInlineSnapshot(`
		{
		  "@@observable": [Function],
		  "dispatch": [Function],
		  "getState": [Function],
		  "replaceReducer": [Function],
		  "subscribe": [Function],
		}
	`)
})
