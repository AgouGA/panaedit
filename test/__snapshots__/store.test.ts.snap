// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should be able to create a store 1`] = `
{
  "@@observable": [Function],
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
}
`;
